Dockerfile

```
@@ -8,8 +8,12 @@ WORKDIR /app
 
 COPY . .
 
+RUN apt update && \
+    apt install openssh-client -y && \
+    mkdir ~/.ssh && \
+    ssh-keyscan github.com >> ~/.ssh/known_hosts
 # Precompile tmail flutter
-RUN bash prebuild.sh
+RUN --mount=type=ssh,required=true ./prebuild.sh
```

```suggestion
RUN --mount=type=ssh,required=true bash prebuild.sh
```
It should be prefixed with bash. Else you can add execute permission to the script and commit the change:
```
chmod +x prebuild.sh
```